pkg/network/render.go

```diff
@@ -185,9 +213,9 @@ func RenderAdditionalNetworks(conf *netv1.NetworkConfigSpec, manifestDir string)
 		return nil, errors.Errorf("invalid Additional Network Configuration: %v", errs)
 	}
 
-	// render Multus when additional networks is provided
+	// render the CRD when additional networks are provided
 	if len(ans) > 0 {
-		objs, err := renderMultusConfig(manifestDir)
+		objs, err := renderAdditionalNetworksCRD(manifestDir)
```

Comment:
Rendering of CRD and CR may fail in a single run, some details [here](https://github.com/openshift/cluster-network-operator/pull/53#issuecomment-452312576)

Reply:
@dcbw I still see this failure with the latest version of PR, not sure if it's specified to how I run the test (run the binary of cluster-network-operator in one of the master nodes). did you ever hit this issue?